src/tools/run-make-support/src/llvm.rs

```diff
@@ -61,6 +61,22 @@ impl LlvmReadobj {
         Self { cmd }
     }
 
+    /// Specify the format of the ELF information.
+    ///
+    /// Valid options are `LLVM` (default), `GNU`, and `JSON`.
+    pub fn elf_output_style(&mut self, style: &str) -> &mut Self {
```

Comment:
An enum would be better for the possible styles, but that may be overkill.

Reply:
In general, we just use loosely typed APIs to avoid the verbosity, the command will fail anyway if you specify an invalid option.